woocommerce/woocommerce-android
# [Product Selector] Analytics events update

### [Product Selector] Analytics events update

Closes: #8624 


### Description

1. Added `product_count` property to `ORDER_PRODUCT_ADD` event
2. Enabled tracking Product Selector actions: 
* product/variation item selected action
* product/variation item unselected action
* "Done" button taps
3. Added new events for tracking the above actions during Order creation flow:
* `ORDER_CREATION_PRODUCT_SELECTOR_ITEM_SELECTED`
* `ORDER_CREATION_PRODUCT_SELECTOR_ITEM_UNSELECTED`
* `ORDER_CREATION_PRODUCT_SELECTOR_CONFIRM_BUTTON_TAPPED`

### Testing instructions
1. Create new order 
2. Open products picker (+ Add products)
3. Select, unselect items and tap the confirmation button at the bottom of the product selector screen

Verify in logcat that the following events are tracked: 
<img width="2042" alt="Screenshot 2023-03-28 at 15 29 00" src="https://user-images.githubusercontent.com/4527432/228252662-794de70e-06b6-47d3-8cfe-7dab0e3b82cc.png">
